api/src/main/java/com/capitalone/dashboard/service/DashboardRemoteServiceImpl.java

```diff
@@ -56,18 +59,51 @@ public DashboardRemoteServiceImpl(
         this.componentRepository = componentRepository;
     }
 
+    /**
+     * Creates a list of unique owners from the owner and owners requests
+     * @param request
+     * @return List<Owner> list of owners to be added to the dashboard
+     * @throws HygieiaException
+     */
+    private List<Owner> getOwners(DashboardRemoteRequest request) throws HygieiaException {
+        DashboardRemoteRequest.DashboardMetaData metaData = request.getMetaData();
+        Owner owner = metaData.getOwner();
+        List<Owner> owners = metaData.getOwners();
+
+        if (owner == null && owners == null) {
```

Comment:
shouldn't you be checking if the list of owners is not empty? It should be changed to (owner==null && CollectionUtils.isEmpty(owners))